conf/app.routes

```diff
@@ -1,3 +1,5 @@
 # microservice specific routes
 
-GET        /combined-api-definitions             uk.gov.hmrc.apiplatformmicroservice.apidefinition.controllers.ApiDefinitionController.fetchApiDefinitionsForCollaborator(collaboratorEmail: Option[String])
+GET        /combined-api-definitions                                                         uk.gov.hmrc.apiplatformmicroservice.apidefinition.controllers.ApiDefinitionController.fetchApiDefinitionsForCollaborator(collaboratorEmail: Option[String])
```

Comment:
I think we should break up app.routes into routes files for each sub-project much like prod.routes points to app.routes for  the app context (usually as / -> app.routes or some such)

Reply:
If you put each routes file in a resource/conf folder per sub-project I think there is some play config for application.conf to point to it (similar to modules)

Reply:
We have decided to split the routes under conf however we were not sure how to put them into the sub-project.

Reply:
Don't worry then.  Just go with what you have got.